src/app/components/cards/PostFull.jsx

```diff
@@ -441,16 +441,49 @@ class PostFull extends React.Component {
             );
         }
 
+        const jsonld = {
+            '@context': 'http://schema.org',
+            '@type': 'Blog',
+            '@id': `https://${APP_DOMAIN}/@${author}/blog`,
+            blogPost: {
+                '@context': 'http://schema.org',
+                '@type': 'BlogPosting',
+                '@id': `https://${APP_DOMAIN}${content.url}`,
+                headline: content.title,
+                text: content.body,
+                url: `https://${APP_DOMAIN}${content.url}`,
+                keywords: content.category,
+                datePublished: content.created,
+                dateModified: content.created,
+                mainEntityOfPage: `https://${APP_DOMAIN}${content.url}`,
+                image: jsonMetadata.image,
+                publisher: {
```

Comment:
Yes it does make sense that steemit is attributed as the publisher of the blogPosting.

Reply:
The documentation of schema.org/Publisher is unfortunately not consistent with the validation tool. According to the documentation, it should be allowed to be a `Person` but the reality looks like this

![Publisher can not be a Person](https://user-images.githubusercontent.com/2051607/34144063-e4e6f0fa-e4ca-11e7-8651-9002a7865ebb.png)

There are [discussions around this](https://productforums.google.com/forum/#!topic/webmasters/hhpenBFwka4) that might suggest that it's only for AMP, which steemit doesn't have anyway. Google seem to suggest that there's far less mandatory fields for non-AMP: https://developers.google.com/search/docs/data-types/article#non-amp-sd

I'd love to get some feedback from anyone with good experience of SEO with schema.org that could confirm this claim. I'll set the `publisher` attribute to the same as `author` in the meanwhile.